ms/MSSel/test/tMSSelection.run

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+# -----------------------------------------------------------------------------
+# Usage: tMSSelection.run
+# -----------------------------------------------------------------------------
+# This script executes the program tMSSelection to test the
+# MSSelection module.
+#
+# $Id$
+#-----------------------------------------------------------------------------
+
+  #CASADEMO=`echo $CASAPATH | awk '{printf("%s/casacore.git/ms/MSSel/test/",$1)}'`+'/'
+  MS=$CASADEMO"mssel_test_small.ms"
+
+  BINPATH='/home/tara2/sanjay/casa_trunk_pristine/trunk/casacore.git/build/ms/MSSel/test/'
+
+  if [ ${#CASAPATH} = 0 ]
+  then
+     echo "UNTESTED: tMSSelection.run (CASAPATH not defined)"
+     exit 3
+  fi
+
+  function runCmd(){
+      #echo "Test:$1 "$2; 
+      eval "$2"
+      }
+
+  function getTestMS() {
+      tar zxf $testsrcdir/$MS.tgz
+  }
+
+  function cleanup() {
+      \rm -rf $MS
+  }
+
+  # Get the test MS (un-tar it from $testsrcdir).
+  getTestMS;
+
+  runCmd 1 "$casa_checktool tMSSelection ms=$MS baseline='1,2,3'";
```

Comment:
@juliantaylor: While I am fixing tMSSelection, is there anything else that needs changes before this request can be closed?  Please let me know soonish.  I would like to focus on this and close this ASAP.